src/App.tsx

```diff
@@ -1,26 +1,99 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import React, { useEffect, useState } from 'react';
+import { deleteTodos, getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import classNames from 'classnames';
+import { FilterOptions } from './FilterOptions';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filter, setFilter] = useState<FilterOptions>(FilterOptions.ALL);
+  const [errorMessage, setErrorMessage] = useState('');
+
+  useEffect(() => {
+    const loadTodos = async () => {
+      try {
+        const loadedTodos = await getTodos();
+
+        setTodos(loadedTodos);
+      } catch (error) {
+        setErrorMessage('Unable to load todos');
```

Comment:
It will be better to create an enum in different file . You could add the error`s string values to your future enum. And use the enum in the places these strings occur.

Reply:
> It will be better to create an enum in different file . You could add the error`s string values to your future enum. And use the enum in the places these strings occur.

Good point